src/gluonts/distribution/lds.py

```diff
@@ -24,6 +27,11 @@
 from gluonts.support.linalg_util import jitter_cholesky
 
 
+class BoundedParam(NamedTuple):
+    lower_bound: float
+    upper_bound: float
+
+
```

Comment:
Trivial but shouldn't we move this to a general utils module for general usage?

Reply:
yes, but earlier Lorenzo insisted that these things be kept local until we find an occasion to use them somewhere else.